packages/flutter/lib/src/gestures/hit_test.dart

```diff
@@ -24,7 +24,7 @@ abstract class HitTestDispatcher { // ignore: one_member_abstracts
   factory HitTestDispatcher._() => null;
 
   /// Override this method to dispatch events.
-  void dispatchEvent(PointerEvent event, HitTestResult result);
+  void dispatchEvent(PointerEvent event, HitTestResult result, bool propagating);
```

Comment:
we probably can't change this API, that's a pretty serious breaking change.

Reply:
This is the case I was thinking of when I asked about breaking API changes recently. The other options I can see:

1. Add a dispatchPropagatingEvent method to this interface instead of adding the param. Because HitTestDispatcher is used as an interface, I can't automatically add an empty implementation, so that breaks the same number of clients as this approach.
1. Add a new interface with the new method, and then make GestureBinding (and WidgetTester) implement both interfaces. The downside here is that logically the dispatching of propagating events seems like it belongs in the interface that's about dispatching events.

Given that only one non-test class in all of Flutter implemented this interface I assumed it was unlikely that third parties were using it directly, but I could certainly do 2 instead if the breaking concern is bigger than the long-term design cruft of a second interface.